app/views/change_logs/history_texts/_dependant_allowance.html.slim

```diff
@@ -1,3 +1,6 @@
+h3.govuk-heading-s
+  = t("change_logs.dependant_allowance.history.header")
+  .govuk-tag class="govuk-!-margin-left-3" = t("change_logs.tags.mtr")
```

Comment:
Adding in the tags, now seem so simple and obvious. 

I thought we were going to something more involved, like building a helper which would take arguments [title] and [tag] and from that we would build this bit.

Reply:
I had thought about building something more structured, but since we'll be replacing all these partials with something dynamic and database-y in the near future, I thought better to leave it simple for now, since it's all pretty manageable as-is.